Project.toml

```diff
@@ -13,7 +13,7 @@ RandomExtensions = "fb686558-2515-59ef-acaa-46db3789a887"
 SHA = "ea8e919c-243c-51af-8825-aaa63cd721ce"
 
 [compat]
-AbstractAlgebra = "0.43.6"
+AbstractAlgebra = "0.43.9"
 FLINT_jll = "^300.100.100"
```

Comment:
What do you need this bump for?

Reply:
There are various recent changes we made in AA and Nemo that are tightly interlocked and I am worried about combinations we never really tested well being put out there. That said, in the end, *shrug*

Reply:
It's fine for me, let's just keep it in